code/obj/item/storage/bible.dm

```diff
@@ -62,8 +68,10 @@
 							if (S)
 								S.set_up(5, 0, T, null, "#000000")
 								S.start()
-			M.HealDamage("All", heal_amt, heal_amt)
-			if(prob(40))
+			var/heal = do_heal_amt(user)
+			M.HealDamage("All", heal, heal)
+			playsound(src.loc, 'sound/effects/faithbiblewhack.ogg', heal^0.6*3+10, 1, -1)
```

Comment:
Not a fan of this, tying a potentially very loud and supernatural sound effect to a common action like this is going to get spammy fast. Not to mention using audio volume as the sole feedback is bad accessibility.
Please just replace this with some vague chat messages or other less obtrusive feedback.

(also `^` is bitwise AND, you mean `**` for power)

Reply:
admittedly i'd built the sound to be really quiet, like a very gentle backdrop to the sound of the bible whack. though i'm not sure if the impact sound is loud in implementation ATM

Reply:
testing it locally yea it can get a bit spammy. i think having a single volume of like, 10, toggling above a certain faith threshold is sane-r. could also put a cooldown on the sfx. 
i swapped it out for this:
```
				if (!ON_COOLDOWN(src, "faith_sound", 1.5 SECONDS))
					SPAWN(1 DECI SECOND)
						playsound(src.loc, 'sound/effects/faithbiblewhack.ogg', 10, FALSE, -1, (rand(94,108)/100))
```

https://github.com/goonstation/goonstation/assets/6061314/9fdc4271-6fcb-4f7f-a123-811e3ef46a5b


do agree using text output is also good for accessibility.